matter/wifi/rs911x/hal/rsi_hal_mcu_timer.c

```diff
@@ -296,7 +296,7 @@ uint32_t rsi_hal_gettickcount(void)
   // Define your API to get the tick count delay in milli seconds from systic ISR and return the resultant value
   struct rsi_timeval tv1;
   gettimeofday(&tv1, NULL);
-  return (tv1.tv_sec * 1000 + tv1.tv_usec / 1000);
+  return (tv1.tv_sec * CONVERSION_VALUE + tv1.tv_usec / CONVERSION_VALUE);
```

Comment:
What kind of CONVERSION_VALUE? CONVERSION_VALUE for what?
If i try to convert Farenheit to Celcius do I use CONVERSION_VALUE ? :)  Nameing isn't saying what it is used for

Reply:
changed